posts/embracing_startup_agility_in_retail.md

```diff
@@ -0,0 +1,61 @@
+6 Ways to Embrace Startup Agility in Retail
+===========================================
+
+The retail industry has been squeezed on many fronts over the past decade, from Internet giants flexing their technology muscle (and willingness to not always turn a profit) to disrupt markets, to fast-moving startups who are unincumbered by big company process, and who are able to attract strong technology talent.
+
+As companies grow, it is an almost enevitable rite of passage that stricter processes are embedded, decision making can become more centralised, and increasing layers added to the heirarchy. While in some measure these things can help an organisation to scale, they can significantly reduce an organisations agility when it comes to embracing changing commercial landscapes.
```

Comment:
enevitable/inevitable

Reply:
should that be "rite of passage that AS stricter processes"?

Reply:
I wanna say there should be an apostrophe in "organisations agility" but I'm not 100% certain!